godotproject/PlayerTongue.tscn

```diff
@@ -6,7 +6,7 @@
 [ext_resource path="res://sfx/tonguestick.wav" type="AudioStream" id=4]
 
 [sub_resource type="CircleShape2D" id=1]
-radius = 5.0
+radius = 2.62027
```

Comment:
```suggestion
radius = 5.0
```
I think this might have just been adjusted during debugging, should we change it back to the old number?

Reply:
Yeah this was changed when we were debugging the rocket door not sticking, I'll revert it.